development/release.md

```diff
@@ -135,15 +220,42 @@ $ rake apt
 $ rake yum
 ```
 
+### Update Docker images
+
+We update Docker images of [Docker Hub][pgroonga-docker-hub].
+
+Clone [Docker repository][pgroonga-docker-repository] and update Dockerfiles.
```

Comment:
```suggestion
Clone [PGroonga's Docker repository][pgroonga-docker-repository] and update Dockerfiles.
```

Reply:
`Dockerfiles`  appears first time in this description. So I think `the` is needless.

Reply:
```
Clone [PGroonga's Docker repository][pgroonga-docker-repository] and update Dockerfiles in tha repository.
```

I have changed this sentence as above.